app/javascript/__tests__/util/languageUtil.test.ts

```diff
@@ -160,4 +161,13 @@ describe("languageUtil", () => {
       expect(getReservedCommunityType("New Type")).toBe("New Type")
     })
   })
+  describe("emptyIfNotTranslated", () => {
+    it("returns a translated string", () => {
+      expect(emptyIfNotTranslated("listings.unitTypes.2 BR")).toBe("2 Bedroom")
+    })
+
+    it("returns an empty string when no translation", () => {
+      expect(emptyIfNotTranslated("listings.unitTypes.500 BR")).toBe("")
```

Comment:
We want the behavior to be to show whatever salesforce is sending if we don't have a translation for the value. So we don't require a code change if a new type of unit is added